master/docs/guides/svn.rst

```diff
@@ -0,0 +1,92 @@
+=================================
+Subversion Version Control System
```

Comment:
I considered at least two other guides:

* git (various practices specific for git related components)
* github (changes, status, maybe something else)

In the light of these ideas, a guide for SVN looked logical.  If I split all these (and following guides) into really small parts, then it's gonna be something else...